bindata/oauth-apiserver/ds.yaml

```diff
@@ -57,7 +57,7 @@ spec:
               --etcd-keyfile=/var/run/secrets/etcd-client/tls.key \
               --etcd-certfile=/var/run/secrets/etcd-client/tls.crt \
               --shutdown-delay-duration=3s \
-              --v=2
+              ${FLAGS}
```

Comment:
I think in the other repo we dealt with this using a specific replace, not a general flags

Reply:
for the `--v` flags we did, replace or append.

Reply:
But this is a replace if I understand the code right, just with a clever string replace syntax `${FLAGS}`. So this is what I expected.

Reply:
yes, I do use string replace and append `--v` to all flags.